api/v1beta2/types.go

```diff
@@ -268,24 +268,16 @@ type PortRange struct {
 	MinimumPort int64 `json:"minimumPort,omitempty"`
 }
 
-// SecurityGroup defines a VPC Security Group that should exist or be created within the specified VPC, with the specified Security Group Rules.
+// VPCSecurityGroup defines a VPC Security Group that should exist or be created within the specified VPC, with the specified Security Group Rules.
 // +kubebuilder:validation:XValidation:rule="has(self.id) || has(self.name)",message="either an id or name must be specified"
-type SecurityGroup struct {
+type VPCSecurityGroup struct {
 	// id of the Security Group.
 	// +optional
 	ID *string `json:"id,omitempty"`
 
-	// name of the Security Group.
-	// +optional
-	Name *string `json:"name,omitempty"`
```

Comment:
@dharaneeshvrd, is this by mistake? you intended to delete VPC and ResouceGroup right

Reply:
Yes, corrected, thanks @Amulyam24 !